website/content/docs/platform/k8s/vso/sources/vault/index.mdx

```diff
@@ -305,6 +307,30 @@ The Vault Secrets Operator can optionally cache Vault client information such as
 
 The [Encrypted client cache](/vault/docs/platform/k8s/vso/sources/vault/client-cache) guide will walk you through the steps to enable and configure client cache encryption.
 
+## Instant updates <EnterpriseAlert inline="true" />
+
```

Comment:
Looks like there was a Tip box [added here recently](https://github.com/hashicorp/vault/blob/357b2949e35bbd230ddb7496aa346e772d8f0017/website/content/docs/platform/k8s/vso/sources/vault/index.mdx?plain=1#L382-L386) too, not sure if we want to include it here:

```suggestion
<Tip title="Feature availability">

  VSO v0.8.0

</Tip>

```